pygsti/tools/optools.py

```diff
@@ -450,20 +450,51 @@ def entanglement_fidelity(a, b, mx_basis='pp'):
         The basis of the matrices.  Allowed values are Matrix-unit (std),
         Gell-Mann (gm), Pauli-product (pp), and Qutrit (qt)
         (or a custom basis object).
+        
+    is_tp_flag : bool, optional (default None)
```

Comment:
I originally didn't like the dual purpose of one variable to denote (i) whether to do the check and (ii) the value in lieu of the check, but I understand why you've done it this way. I think I'll be OK with it, so long as this docstring is a little more clear. Maybe something like:

Flag indicating both matrices are TP. If None (the default), an explicit check is performed. If True/False, the check is skipped and provided value is used (faster, but should only be used when the user is certain this is true apriori).

IDK exactly, but I want it to be clear that None is not just a dummy default - it actually changes the behavior of the function.

Reply:
Great suggestion, wording was definitely unclear. I more or less verbatim copied your language directly since I thought it was very clear and concise.